__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+	return seed(testData);
+});
+
+afterAll(() => {
+	db.end();
+});
+
+describe("Get", () => {
+	describe("api/categories", () => {
+		test("200-responses with an array of objects", () => {
+			return request(app)
+				.get("/api/categories")
+				.expect(200)
+				.then((result) => {
+					expect(result.body.length).toBe(4);
+					result.body.forEach((categories) => {
```

Comment:
to prevent this being result.body - we can destructure in the test like 
(({ body } => and when we send categories over  in the **controller** we can do so by putting categories in object brackets {categories} and this will make the code a bit clearer when accessing the data